### PR TITLE
Chachacha deployment

### LINKED ISSUE #68: [CI] Add Docker builds for commits added to the main branch
We should add a CI job to build and publish Docker images on new commits to the main branch.

### PR DESCRIPTION
**Pull request by c410-f3r**
Also fixes #68 

CI will deploy a parachain on Chachacha everytime a commit is pushed to the `chachacha` branch, preferably when re-basing from `main`.

```
PRs ------>  main ------> chachacha
             |            |
             |            ------> Deploy to Chachacha
             |
             -----> Push images to Docker Hub
```

### CODE REVIEW

File: .github/workflows/chachacha.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Publish
+        uses: appleboy/ssh-action@master
+        with:
+          host: 45.33.117.205
+          key: ${{ secrets.CHACHACHA_VM_PRIVATE_KEY }}
+          passphrase: ${{ secrets.CHACHACHA_VM_PASSWORD }}
```
Comment (c410-f3r) [inline .github/workflows/chachacha.yml:23]: It is necessary to provide CHACHACHA_VM_PRIVATE_KEY and CHACHACHA_VM_PASSWORD secrets

File: .github/workflows/docker-hub.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
```
Comment (c410-f3r) [inline .github/workflows/docker-hub.yml:34]: More secrets for image deployment

File: .github/workflows/chachacha.yml (hunk 1 above)
Comment (lsaether) [reply .github/workflows/chachacha.yml:23]: secret has been added for the `ci` user on the vm

File: .github/workflows/docker-hub.yml (hunk 2 above)
Comment (lsaether) [reply .github/workflows/docker-hub.yml:34]: added

**Pull request closed without merging**